Components/HeroPostHeader/functions.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+namespace Flynt\Components\HeroPostHeader;
+
+use Timber\Timber;
+use Flynt\Features\Components\Component;
+use Flynt\Utils\Asset;
+
+add_action('wp_enqueue_scripts', function () {
+    Component::enqueueAssets('HeroPostHeader', []);
+});
+
+add_filter('Flynt/addComponentData?name=HeroPostHeader', function ($data) {
+    $data['post'] = Timber::get_post();
+    $data['providers'] = getShareProviders(
+        $data['post']->post_title,
+        $data['post']->link,
+        $data['post']->post_excerpt
+    );
+    return $data;
+});
+
+function getShareProviders($postTitle, $postUrl, $postExcerpt)
+{
+    $providers = [
+        [
+            'title' => 'Facebook',
+            'class' => 'share-facebook',
+            'url' => 'https://www.facebook.com/sharer/sharer.php?u=$url'
+        ],
+        [
+            'title' => 'Twitter',
+            'class' => 'share-twitter',
+            'url' => 'https://twitter.com/home?status=$url'
```

Comment:
This does not work on a real mobile device. It will redirect to mobile.twitter.com. The share url now needs to be like this: `https://twitter.com/intent/tweet?url=$url`